benefits/static/css/styles.css

```diff
@@ -173,22 +176,23 @@ main {
   --footer-link-color: #73b3e7;
   --footer-link-width: 100%;
   --footer-link-hover-color: #9b74d7;
-  --footer-link-font-weight: 500;
+  --footer-link-font-weight: var(--medium-font-weight);
```

Comment:
If you're going to rebase on `dev`, you want to apply this variable to the new `#previous-page-button` style I added? It uses the same font weight.